internal/oem/oem.go

```diff
@@ -103,10 +105,129 @@ func init() {
 		provider: gce.Creator{},
 		config: types.Config{
 			Systemd: types.Systemd{
-				Units: []types.SystemdUnit{{
-					Name:   "coreos-metadata-sshkeys@.service",
-					Enable: true,
-				}},
+				Units: []types.SystemdUnit{
+					{
+						Name:   "coreos-metadata-sshkeys@.service",
+						Enable: true,
+					},
+					{
+						Name:   "google-accounts-manager.service",
+						Enable: true,
+						Contents: `
+[Unit]
```

Comment:
I'm not sure how this would work. If the units were installed to /usr/share/oem, how would we enable them? Right now, Ignition doesn't have the ability to copy files (we could add it though).

While it is unfortunate that this info in scattered across repos, I'm not sure it's the worst thing. Ignition already knows a lot about the specific platform, and this is just a matter of moving the configs completely from coreos-overlay to Ignition. It's still a little split with GCE, but I plan on cleaning that up before this ships.